lib/kittikrb/cursor.rb

```diff
@@ -115,6 +146,113 @@ def reset_tty!
         self
       end
 
+      def erase(x1, y1, x2, y2)
```

Comment:
We need comment above this method, because there is not obvious argument list (actually, not obvious method name, but we can't change it for API compatibility).

```
   # Erase the specified region.
   # The region describes the rectangle shape which need to erase.
```